### PR TITLE
Minor Build Improvements

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Comment by gabemontero** [comment]
looks OK to me

@dwalluck - you have any comments / questions ?

File: java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java [hunk 1]
```diff
@@ -133,7 +138,9 @@ private void doBuildAnalysis(String scmUrl, String scmTag, String context, Build
                     }
                     info.invocations.add(
                             new ArrayList<>(List.of("clean", "install", "-DskipTests", "-Denforcer.skip", "-Dcheckstyle.skip",
-                                    "-Drat.skip=true", "-Dmaven.deploy.skip=false", "-Dgpg.skip")));
+                                    "-Drat.skip=true", "-Dmaven.deploy.skip=false", "-Dgpg.skip",
+                                    "-Dmaven.javadoc.failOnError=false", "-Dmaven.javadoc.failOnError=false",
+                                    "-Dmaven.javadoc.failOnWarnings=false")));
```
Comment (dwalluck) [inline java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java:143]: I thought that generally if it got a javadoc error, it was the case of using too new a JDK (11 vs. 8, for example).

Comment (dwalluck) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java:143]: Specifically, I saw a complaint about the `<tt>` tag under newer JDKs.

Comment (stuartwdouglas) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java:143]: Hmm,looks like you are right. I did this to get the CDI  build to pass but it looks like the JDK8 build is failing for  a different reason.

Comment (dwalluck) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java]: I think it's a case of not specifying plugin versions and getting something too new.

Comment (stuartwdouglas) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java]: Hopefully I will have a solution for that soon.

**Pull request merged**